### PR TITLE
build(node): drop node 8, support node 10.13+, node 12 

### PR DESCRIPTION
**Pull request by sgtcoolguy**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-27605

### CODE REVIEW

**Review by cb1kenobi — APPROVED** [review]
Yes!

File: package.json [hunk 1]
```diff
@@ -153,10 +153,10 @@
     "visual studio": ">=10 <=12",
     "windows phone sdk": "8.x",
     "msbuild": ">=4.0",
-    "node": "8.x || 10.x"
+    "node": "10.x || 12.x"
   },
   "engines": {
-    "node": ">=8"
+    "node": ">=10.13"
```
Comment (cb1kenobi) [inline package.json]: Just out of curiosity is there a technical reason 10.12 won't work? Could/should this be `10.x`? I know the daemon requires specifically 10.2.0 or newer, but what is it with 10.13.0?

Comment (sgtcoolguy) [reply package.json]: Spoke with @mukherjee2 about this on Teams. My understanding is that the daemon requires 10.2+, but we might as well use Node's own LTS definition. 10.13 was the first official LTS release on the Node 10.x line.

Comment (cb1kenobi) [reply package.json]: OK, fine by me.

**Pull request merged**